src/Tribe/Asset/Select2.php

```diff
@@ -21,9 +21,8 @@ public function handle() {
 		// we know of other plugins loading a version of select2 compatible with our needs
 		// let's not queue the script twice.
 		if ( ! $this->has_script_alias( 'select2' ) ) {
-			$script_handle = $this->prefix . '-select2';
-			wp_enqueue_script( $script_handle, $path, 'jquery', '3.2' );
-			Tribe__Events__Template_Factory::add_vendor_script( $script_handle );
+			wp_enqueue_script( 'select2', $path, 'jquery', '3.2' );
+			Tribe__Events__Template_Factory::add_vendor_script( 'select2' );
```

Comment:
I have no idea what you are talking about; the handle I've changed as `events-recurrence.js` would not q due to it, it does with this modification.
4.x what?